app/controllers/admin/feedback_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Admin::FeedbackController < Admin::ApplicationController
+class Admin::FeedbackController < SuperAdmin::ApplicationController
```

Comment:
@despo does this mean we don't want to give organizers access to feedbacks?

Reply:
They can still access the feedback that has been given for their chapter specifically.

Reply:
@despo ok, cool